tests/framework/installer/install_helper.go

```diff
@@ -158,7 +159,7 @@ func (h *InstallHelper) CreateK8sRookCluster(clusterNamespace string) (err error
 }
 
 //InstallRookOnK8s installs rook on k8s
-func (h *InstallHelper) InstallRookOnK8s(clusterNamespace string) (err error) {
+func (h *InstallHelper) InstallRookOnK8s(clusterNamespace string) (result bool, err error) {
```

Comment:
i wouldn't generally use named return values. In the header, just declare `(bool, error)`. Then you can be explicit everywhere about the values to return